capitulo 1/lang_en.json

```diff
@@ -4237,161 +4237,161 @@
   "scr_text_slash_scr_text_gml_2304_0": "\\C3 ",
-  "scr_text_slash_scr_text_gml_879_0": "Abrir#Geladeira",
-  "scr_text_slash_scr_text_gml_880_0": " #Não",
+  "scr_text_slash_scr_text_gml_879_0": "#Abrir",
+  "scr_text_slash_scr_text_gml_880_0": " #Não Abrir",
   "scr_text_slash_scr_text_gml_881_0": "Ver fotos",
   "scr_text_slash_scr_text_gml_902_0": "* Nada aconteceu./%",
   "scr_text_slash_scr_text_gml_947_0": "* (Uma foto de sua mãe e pai no dia do casamento.)/",
-  "scr_text_slash_scr_text_gml_948_0": "* (Ela está segurando um buque com 7 flores.)/",
-  "scr_text_slash_scr_text_gml_949_0": "* (Um monstro de aparência de rena fica nas proximidades em um smoking.)/",
-  "scr_text_slash_scr_text_gml_950_0": "* (Eles parecem felizes.)/%",
-  "scr_text_slash_scr_text_gml_894_0": "* (É uma geladeira velha com algumas fotos nela.)/",
+  "scr_text_slash_scr_text_gml_948_0": "* (Ela está segurando um buquê com sete flores.)/",
+  "scr_text_slash_scr_text_gml_949_0": "* (Um monstro parecido com uma rena está próximo em um smoking.)/",
+  "scr_text_slash_scr_text_gml_950_0": "* (Todos parecem felizes.)/%",
+  "scr_text_slash_scr_text_gml_894_0": "* (É uma geladeira velha com algumas fotos.)/",
   "scr_text_slash_scr_text_gml_3093_0": "\\C3 ",
-  "scr_text_slash_scr_text_gml_919_0": "* (Tudo que tem dentro é um pote de picles.)/",
+  "scr_text_slash_scr_text_gml_919_0": "* (Tudo que tem dentro é um pote com um picles dentro.)/",
   "scr_text_slash_scr_text_gml_914_0": "* (Você decide não comer)/%",
   "scr_text_slash_scr_text_gml_943_0": "* (Você decide não olhar.)/%",
-  "scr_text_slash_scr_text_gml_912_0": "* (Tudo que tem dentro é um pote de picles...)/",
-  "scr_text_slash_scr_text_gml_913_0": "* (... e^1, por algum motivo^1, um ovo.)/",
+  "scr_text_slash_scr_text_gml_912_0": "* (Tudo que tem dentro é um pote com um picles dentro...)/",
+  "scr_text_slash_scr_text_gml_913_0": "* (... e^1, por algum motivo^1, um ovo solitário.)/",
   "scr_text_slash_scr_text_gml_920_0": "* (Você decide não comer.)/%",
-  "scr_text_slash_scr_text_gml_924_0": "* (Tudo que tem dentro é um pote de picles.)/",
+  "scr_text_slash_scr_text_gml_924_0": "* (Tudo que tem dentro é um pote com um picles dentro.)/",
   "scr_text_slash_scr_text_gml_2852_0": "* (...)/",
-  "scr_text_slash_scr_text_gml_926_0": "* (Você põe o Ovo que estava carregando^1, também...)/%",
-  "scr_text_slash_scr_text_gml_907_0": "* (Tem dois OVOS dentro da geladeira.)/%",
-  "obj_readable_room1_slash_Other_10_gml_1212_0": "* (É uma TV pequena.)&* (No topo tem alguns quadrinhos de super-herois.)/%",
+  "scr_text_slash_scr_text_gml_926_0": "* (Você põe o Ovo que você estava carregando dentro^1, também...)/%",
+  "scr_text_slash_scr_text_gml_907_0": "* (Tem dois Ovos dentro da geladeira.)/%",
+  "obj_readable_room1_slash_Other_10_gml_1212_0": "* (É uma TV pequena.)&* (No topo tem alguns quadrinhos de super heróis.)/%",
```

Comment:
hã-harãm super-heróis é com hífen
https://webinformado.com.br/tag/super-heroi-junto-ou-separado/